Development-Guide/Compile-and-Flash-STMBoard-Guide/README.adoc

```diff
@@ -0,0 +1,47 @@
+== *Compiling and Flashing STM Board*
+
+----
+This has been tested on Linux Mint and is expected to work on other Debian based distros
+----
+
+
+=== Setup
+. Go to UCSolarCarTeam repo > Recruit Resources, and clone
+. Navigate to STM32F4Discovery-Resources and run the auto installer `sudo ./install.sh`. This will download STLink, gcc4mbed, and CubeMX2Makefile on your computer and add gcc4mbed to your path
+. Use a text editor you're comfortable with to add STLink and CubeMX2Makefile to your path. You can usually edit it in your .profile. For example: `nano ~/.profile`
+. Add them by copying  `export PATH=$PATH:/opt/CubeMX2Makefile` and `export PATH=$PATH:/opt/stlink` in your profile.
+. Save, exit, and source your profile `source ~/.profile`
+. Go to the link: http://www.st.com/en/development-tools/stm32cubemx.html
```

Comment:
make it `Go to the download page of the official ST website` and then create a compressed link using that text, having full links in documents looks bad generally.